src/components/Layout/Layout.tsx

```diff
@@ -214,87 +218,93 @@ export const Layout = ({
                 currentSite="Docs"
                 isGen2={isGen2}
               />
-              <Flex className={`layout-search layout-search--${pageType}`}>
-                <Button
-                  onClick={() => handleMenuToggle()}
-                  size="small"
-                  ref={menuButtonRef}
-                  className="search-menu-toggle mobile-toggle"
-                >
-                  <IconMenu aria-hidden="true" />
-                  Menu
-                </Button>
+              <View as="header" className="layout-header">
```

Comment:
We can wrap the search and sidebar in a `<header>` so that they're part of a landmark.